---
odom_transform_node:
  ros__parameters:
    odom_topic: t265_camera/odom
    pose_topic: t265_camera/odom/to_base
    target_frame: base_link
ekf_filter_node:
  ros__parameters:
    frequency: 30.0
    sensor_timeout: 0.5
    two_d_mode: true
    transform_time_offset: 0.0
    transform_timeout: 0.0
    print_diagnostics: true
    debug: false
    debug_out_file: efk_log.txt
    publish_tf: true
    publish_acceleration: false
    map_frame: map
    odom_frame: odom
    base_link_frame: base_link
    world_frame: odom
    # odom if fusing odometry, set to map if fusing global position data

    use_control: true
    stamped_control: false
    control_config: [true, false, false, false, false, true]

    # x,     y,      z,
    # roll,  pitch,  yaw,
    # vx,    vy,     vz,
    # vroll, vpitch, vyaw,
    # ax,    ay,     az

    pose0: /t265_camera/odom/to_base
    pose0_config: [true,  true,  false,
                   false, false, true,
                   false, false, false,
                   false, false, false,
                   false, false, false]
    pose0_differential: false
    pose0_relative: true

    pose1: /laser_pose
    pose1_config: [true,  true,  false,
                   false, false, true,
                   false, false, false,
                   false, false, false,
                   false, false, false]
    pose1_differential: true
    pose1_relative: false

    odom0: /odom_wheel
    odom0_config: [false, false, false,
                   false, false, false,
                   true,  false, false,
                   false, false, true,
                   false, false, false]
    odom0_differential: false
    odom0_relative: false
